src/main/java/org/moskito/control/ui/action/inspection/ShowThresholdsAction.java

```diff
@@ -38,8 +38,13 @@ public ActionCommand execute(ActionMapping mapping, FormBean formBean, HttpServl
         if (application == null) {
             return mapping.error();
         }
-        Component component = application.getComponent(componentName);
```

Comment:
getComponent(String) method never return null. It throws IllegalArgumentException instead